src/server/rpc/procedures/water-watch/water-watch.js

```diff
@@ -16,7 +16,14 @@ function encodeQueryData(options) {
     return ret.join('&');
 }
 
-
+/**
+ * Get the water data for sites within a bounding box.
+ * @param {Latitude} minLatitude Minimum latitude of bounding box
+ * @param {Latitude} maxLatitude Maximum latitude of bounding box
+ * @param {Longitude} minLongitude Minimum longitude of bounding box
+ * @param {Longitude} maxLongitude Maximum longitude of bounding box
+ * @returns {Promise} Messages sent of data
```

Comment:
I would annotate the return value of the promise - not the promise itself - since this is for the end netsblox user and they will not notice receive the promise.

Reply:
It also might be useful to use more precise terminology for this and (optionally) add this link: https://help.waterdata.usgs.gov/tutorials/surface-water-data/how-do-i-interpret-gage-height-and-streamflow-values somewhere.

Reply:
Are the return values shown anywhere? I was just putting that in for completeness, but they don't show up in the help dialog.

Reply:
Not currently, so it isn't a huge deal. However, it would be good to make this info available to the users and `Promise` isn't very helpful. One compromise would be to use the standard JS approach when documenting promises and include the type `Promise<String>` or something

Reply:
Is there a better way to include a link? Just having it as text isn't very convenient.

Reply:
If we used the latter option, it could be more obvious to the developers on netsblox, too. The server would just need to strip out the `Promise` portion when generating the documentation.

Reply:
Made an issue and made the promises more specific.